packages/core/src/lib/utils.ts

```diff
@@ -155,7 +170,8 @@ const skipTransformation = (
   msg: string
 ) => {
   // TODO: Add better error reporting
-  api.report(msg)
+  console.log(msg); // <== commented out api.report and used console.log for error logging
```

Comment:
I changed the api. sync with staging and fix the make sure to fix the merge conflict